src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -16,6 +16,21 @@
 short-summary: Manage web apps.
 """
 
+helps['webapp auth'] = """
+    type: group
+    short-summary: Manage webapp authentication and authorization
+"""
+
+helps['webapp auth show'] = """
+    type: command
+    short-summary: Show the authentification settings for the webapp.
+"""
+
+helps['webapp auth update'] = """
+    type: command
+    short-summary: Update the authentication settings for the webapp.
+"""
```

Comment:
Suggest add an example to show how to enable AAD. You can add more later to include how to use facebook, and twitter